tests/Datadog.Sketches.Tests/Datadog.Sketches.Tests.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net461;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
```

Comment:
I guess you _may_ want to test other runtimes too, (esp .net5/.net6), just to be safe? No need to build for them, just testing is fine

Reply:
Fair